### ISSUE TITLE
Ctrl-Shift-S still failing

### ISSUE THREAD

**Issue by Hong-Revo** [MikhailArkhipov/vscode-r#15]
Looks like a quoting issue, there are extraneous "s in the `source` call. This is what shows up in the R terminal.

```
source(""c:/Users/hongooi/OneDrive - Microsoft/AzureStor/misc/blob_internal.R"")
```

**Comment by MikhailArkhipov** [comment]
Looks like specific to paths with whitespaces which already come quoted. Besides the fact that Unicode paths disappear since R is not Unicode on Windows.

![image](https://user-images.githubusercontent.com/12820357/52538407-145b1a00-2d27-11e9-8b42-a713ef1ca55b.png)
![image](https://user-images.githubusercontent.com/12820357/52538460-ac590380-2d27-11e9-909c-8bbdf15e09cc.png)


![image](https://user-images.githubusercontent.com/12820357/52538419-36549c80-2d27-11e9-8ecc-33fd64bec2ec.png)

**Comment by MikhailArkhipov** [comment]
Fixed in 0.0.6
